code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -183,13 +188,23 @@
 		M.Dizzy(5)
 		if(iscultist(M) && prob(5))
 			M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
+		else if(is_servant_of_ratvar(M) && prob(5))
+			switch(pick("speech", "message", "emote"))
+				if("speech")
+					M.say("...[pick("Ratvar... lbhe yvtug tebjf qnex", "Jurer ner lbh, znfgre?", "Ur yvrf ehfgvat va Reebe", "Chetr nyy hagehguf naq... naq... fbzrguvat")]...")
+				if("message")
+					M << "<span class='warning'><b>[pick("Ratvar's illumination of your mind has begun to flicker.", "He lies rusting in Reebe, derelict and forgotten. And there he shall stay.", \
+					"You can't save him. Nothing can save him now.", "It seems that Nar-Sie will triumph after all.")]</b></span>"
```

Comment:
It's all a matter of perspective. The most terrifying villain there is, is one that thinks what he is doing is the right thing.
That's what Ratvarian followers are. They aim to free ratvar and stop narsie at all costs, anyone who interferes in their task is attempting to stop their holy mission. They are trying to save the world from narsie in their eyes. Anyone stopping them is playing into narsie's hands.